### DISCUSSION TITLE
Allow basic image editing like crop after upload?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13294] [Feature Requests]
Nothing fancy, just to allow the user to do things like a simple crop by dragging out a frame etc.

Pimcore has a really nice feature that allows you to define multiple 'points of interest' on an image where you can crop around them, which may be a nice idea too moving forwards when retrieving via the api.

**Comment by benhaynes** [comment]
This exists: https://docs.directus.io/app/file-library/#edit-an-image

@erondpowell that docs gif has an oddly small edit preview image... can you confirm that's still happening on your side? We should regenerate the video.

**Reply by sxsws** [reply]
ohh thanks. it's accessible after upload? I was thinking between the file chooser and the actual upload. I didn't realise it was accessible from the files area because I clicked on edit on the image in an item and just accessed the actual image data rather than edit the image!

**Reply by LasseRosenow** [reply]
Btw. Somewhat related to #3286

**Comment by erondpowell** [comment]
It appears to be a bug causing that @benhaynes 
I am fixing docs now and opening a bug issue too

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
